docs/testbook.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Generate the testbook for the functional tests
+#
+# To run this file you will need testbook installed into your DOAJ env
+#
+# source [doaj env]/bin/activate
+# git clone git@github.com:CottageLabs/testbook.git
+# cd testbook
+# pip install -r requirements.txt
+#
+# ~~Testbook:Script->Testbook:Technology~~
+
+BRANCH=$(git branch 2>/dev/null | grep '^*' | colrm 1 2)
+
+TESTBOOK_URL="https://doaj.github.io/doaj-docs/$BRANCH/testbook"
+DOAJ_URL="http://testdoaj.cottagelabs.com/"
```

Comment:
I suppose there's no easy way to set this to a particular instance, e.g. reading from dev.cfg (which is _almost_ bash compatible) since we might be generating for live from localhost, for instance.

Production might be a better default, though, or have an env switch. e.g. `./testbook.sh dev|test|live`

Reply:
If you want a switch, that already exists, you can just use `./testbook -t https://testdoaj2.cottagelabs.com`

I'm assuming that we don't want to ever point users at the live site for functional testing, so I don't think doaj.org is the right default